uber/templates/summary/index.html

```diff
@@ -2,120 +2,61 @@
 {% block title %}Registration Stats{% endblock %}
 {% block content %}
 
-<b>Attendees checked in:</b> {{ checkin_count }}
```

Comment:
The actual content of this page is mostly unchanged, but I changed the formatting a bit and how it's rendered.